Makefile

```diff
@@ -0,0 +1,14 @@
+#
```

Comment:
What is the Makefile for? Homebrew?

Reply:
Yea, its generated by FGo. I'll repost the link here:

https://jkawamoto.github.io/fgo/